test_app/tests/authentication/views/test_authenticator_map.py

```diff
@@ -84,6 +85,9 @@ def test_authenticator_map_invalid_map_type(admin_api_client, local_authenticato
     assert '"invalid" is not a valid choice.' in response.data['map_type'][0]
 
 
+@pytest.mark.skipif(
+    connection.vendor == 'sqlite', reason='PositiveIntegerField min_value was not discovered but fixed in Django 5 https://github.com/django/django/pull/16601'
```

Comment:
Can we also check `django.VERSION[0] < 5`, so this test works whenever we upgrade?

Reply:
I was just being lazy. I'll push a commit.